app/page.tsx

```diff
@@ -54,6 +55,163 @@ export default function Airdrop() {
         </div>
         {isUser && <Footer />}
       </div>
+  */}
+
+      <div className={`flex-col flex justify-start items-center bg-cover bg-no-repeat bg-bottom page-wrapper`}>
+        <div className="page-wrapper-bg">
+          <div className="page-wrapper-bg page-wrapper-bg-layer1"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-layer2"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-layer3"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-layer4"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-layer5"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-stars1"></div>
+          <div className="page-wrapper-bg page-wrapper-bg-stars2"></div>
+        </div>
+          <Topbar />
+              <div className="w-full flex flex-col items-center">
+                <AnimatePresence>
+                  {currentComponent === "landing" && <Landing />}
+                  {currentComponent === "signup" && <SignUp />}
+                  {currentComponent === "dashboard" && <Dashboard />}
+                </AnimatePresence>
+              </div>
+            {isUser && <Footer />}
+        
+      </div>
+
+      <style>
```

Comment:
@creatix please use "Tailwind" as much as you can, instead of CSS.

And if there are some CSS left which is challenging to implement in Tailwind, then use "styles folder", and move CSS code inside /styles/globals.css file or create a new css file:
https://github.com/fuseio/airdrop-dapp/tree/master/styles

Reply:
Could you do that please? i don't have experience with Tailwind and i don't have time to work on it

Reply:
@creatix sure